src/lua/compat.lua

```diff
@@ -210,6 +218,16 @@ local options = {
         brief = BOX_SPACE_MAX_BRIEF,
         action = tweak_action('BOX_SPACE_MAX', 2147483647, 2147483646)
     },
+    box_error_serialize_verbose = {
+        default = 'new',
```

Comment:
A window for breaking changes was closed in Dec, 2023 with the 3.0.0 release. Now the only way to go with the change is to make the new behavior disabled by default.